examples/rhai-test.config.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
So one problem with moving this in here is this breaks the development flow of running `cargo run` and may also break the github actions 😓

Reply:
Thinking having a copy in the root for development still in addition to this copy in examples is the way to go. :)

Reply:
Oh, I see. I was making it easier to run the examples after the installer is added. Let me add it back for now